torch/onnx/_patch_torch.py

```diff
@@ -12,7 +12,7 @@
 def _graph_op(
     g: torch._C.Graph,
     opname: str,
-    *raw_args: torch._C.Node,
+    *raw_args: torch._C.Value,
```

Comment:
should it be a tuple of values instead? (from the star?

Reply:
We can (need to) omit the tuple: https://stackoverflow.com/questions/37031928/type-annotations-for-args-and-kwargs